tools/sz_repo_cli/lib/src/commands/src/deploy_web_app_command.dart

```diff
@@ -77,10 +77,10 @@ class DeployWebAppCommand extends Command {
     // easier.
     isVerbose = true;
 
-    final googleApplicationCredentialsFile = _parseCredentialsFile(argResults);
+    final googleApplicationCredentialsFile = _parseCredentialsFile(argResults!);
 
-    final overriddenDeployMessageOrNull = _parseDeployMessage(argResults);
-    final releaseStage = _parseReleaseStage(argResults);
+    final overriddenDeployMessageOrNull = _parseDeployMessage(argResults!);
+    final releaseStage = _parseReleaseStage(argResults!);
```

Comment:
Since we can now infer by the types that `overridenDeployMessageOrNull` can be `null`, we could remove the "orNull". What do you think?

Reply:
Yes makes sense 👍